docs/developer/procedures/release.rst

```diff
@@ -136,14 +136,14 @@ When creating the first release candidate of a series, there are some extra step
     git commit -m "Change 28.x to stable branch"
     git push upstream 28.x
 
-  For the new `maintenance` branch, and the remote for the official GeoTools is called ``upstream``::
+  For the new `maintenance` branch (which was the previous `stable` branch)::
   
-    git checkout 26.x
```

Comment:
@jodygarnett please check that these changes below are correct.

If 27.x was the stable branch, the new stable branch is 28.x (set above) and then 27.x should become the maintenance branch.  Does the 26.x branch ever get set to archive or similar?

Reply:
yes, I think that happens further down the page.

Reply:
Suggested fix: https://github.com/geotools/geotools/pull/4907